src/CommaSeparatedJsonParser.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+/**
+ * Class CommaSeparatedJsonParser
```

Comment:
Could we remove the `Class ...` and `@package ..`? :) Just to keep it consistent with the other classes